### PR TITLE
Update man 03

### PR DESCRIPTION
**Pull request by les542**

### CODE REVIEW

File: docs/man [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Special files (usually found in /dev)
+File formats and conventions eg /etc/passwd
+Miscellaneous (including macro packages and conventions), e.g. groff(7)
+System administration commands (usually only for root)
+Kernel routines [Non standard]
+
+## Basic command structure
+For all descriptions below, the dollar sign indicates that BASH command prompt.
+
+$ `man [options] [directory path]`
```
Comment (jmf-nyu) [inline docs/man:26]: the second argument in the `man` command structure is the [name of command] not the [directory path]

File: docs/man [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Summary 
+The `man` command in Linux is used to display the user manual of any command that we can run on
+the terminal. It provides a detailed view of the command which includes NAME, SYNOPSIS,
+DESCRIPTION, OPTIONS, EXIT STATUS, RETURN VALUES, ERRORS, FILES, VERSIONS, EXAMPLES,
+AUTHORS and SEE ALSO.
+https://www.nielit.gov.in/gorakhpur/sites/default/files/Gorakhpur/Alevel_unix_27april20_AKM.pdf
+
+Every manual is divided into the following sections:
+
+Executable programs or shell commands
```
Comment (jmf-nyu) [inline docs/man:12]: This list of sections could be presented as bullets by placing an asterisk before each item.

File: docs/man [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# The man Command
+
+## Summary 
+The `man` command in Linux is used to display the user manual of any command that we can run on
+the terminal. It provides a detailed view of the command which includes NAME, SYNOPSIS,
+DESCRIPTION, OPTIONS, EXIT STATUS, RETURN VALUES, ERRORS, FILES, VERSIONS, EXAMPLES,
+AUTHORS and SEE ALSO.
+https://www.nielit.gov.in/gorakhpur/sites/default/files/Gorakhpur/Alevel_unix_27april20_AKM.pdf
```
Comment (jmf-nyu) [inline docs/man:8]: URLs should be correctly formatted as hyperlinks using the Markdown syntax described in the documentation here: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#links

File: docs/man [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+## Possible Flags
+
+### `-a`
+The `-a` flag helps users search and skips to current screen. This option helps us to display all the available intro manual pages in succession.
+See: 
+https://www.geeksforgeeks.org/man-command-in-linux-with-examples/
+
+### `-?`
+The `-?` flag  displays help (from command line).
+See: Terminal
```
Comment (jmf-nyu) [inline docs/man:37]: I'm not sure what this means

**Review by jmf-nyu — CHANGES_REQUESTED** [review]
This is mostly looking good with a few small comments on some of the lines.

Most importantly, you will need to edit the index.md file to include a link to your new page for the man command. That file wasn't edited in this pull request. 

Second, you will need to rename your new file to "man.md". The .md extension makes sure that GitHub pages sees this file as a Markdown file.  

Before you revise your pull request, make sure you are fetching and pulling changes from the class repository. I have already merged a few pull requests.

**Comment by les542** [comment]
Updated the man file to "man.md" to become a markdown file. Did revisions. I forgot to sync the fork beforehand and was concerned about it deleting my changes or 'commits.'

**Review by jmf-nyu — CHANGES_REQUESTED** [review]
You've fixed some of the issues I've mentioned. Almost there. However, there are still a few more. Please check my comments that refer to various lines of your code. 

Also, keep in mind that by not syncing your fork some conflicts were created. They are simple enough that I can fix them when submit the pull request. Just keep that in mind for the future that you always want to be syncing your fork (i.e., fetching new changes and merging them on your copy of the code) before starting new work to avoid conflicts.

File: docs/man (hunk 3 above)
Comment (jmf-nyu) [reply docs/man]: @les542 This hasn't been resolved yet

File: docs/man (hunk 4 above)
Comment (jmf-nyu) [reply docs/man]: Please resolve

**Pull request merged**